test_cases/stdlib/builtins/check_dict.py

```diff
@@ -18,7 +18,7 @@
 
 
 class KeysAndGetItem(Generic[_KT, _VT]):
-    data: dict[_KT, _VT]
+    data: dict[_KT, _VT] = {}
```

Comment:
I'd prefer for this to be initialised in an `__init__` method, so it's 100% unambiguous that it's an instance attribute and not a class attribute